### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by nsantiagoblair**
I want to merge my portfolio page to master. This contains:

-layout
-styling
-I.E. 9 compatibility
-the projects assets folder

Still contains placeholders (lorem ipsum and cat.jpg) for the content, which I'll be working on while this branch is being reviewed.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,475 @@
+* {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+body {
+    color: white;
+    font-family: 'Shadows Into Light', cursive;
+
```
Comment (mporam) [inline styles.css]: whitespace?

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+.ie9 #about main p {
+    color: black;
+}
+
+a {
+    text-decoration: none;
+    color: white;
+    font-family: 'Permanent Marker', cursive;
+
```
Comment (mporam) [inline styles.css]: why space?

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,475 @@
...
+    to { left: 0;}
+}
+
+@keyframes  navitems {
+    from {right: -1000px;}
+    to { right: 0;}
+}
+
+.nav-list a {
+    text-decoration: none;
```
Comment (mporam) [inline styles.css]: dont need this as applied globally to all a's

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,475 @@
...
+    color: white;
```
Comment (mporam) [inline styles.css]: hex?

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,475 @@
...
+    font-size: 1.7rem;
+    padding: 20px;
+}
+
+.nav-list a:hover {
+    text-shadow: 5px 5px 5px gray, -5px -5px 5px gray,
```
Comment (mporam) [inline styles.css]: hex colors?

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,475 @@
...
+                -5px 5px 5px gray, 5px -5px 5px gray;
+}
+
+.nav-list a:hover {
+    text-decoration: underline
+}
+
+.nav-list [class*="under-"] {
```
Comment (mporam) [inline styles.css]: what is this? no html with a class of under-

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,475 @@
...
+    position: relative;
+    width: 20%;
+    margin: 0 auto;
+    z-index: 5;
+}
+
+.home-link:hover {
+    animation: shake 0.5s;
+    animation-iteration-count: infinite;
+    text-shadow: 5px 5px 5px gray, -5px -5px 5px gray,
```
Comment (mporam) [inline styles.css]: hex colors?

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,475 @@
...
+}
+
+.home-link a {
+   font-size: 1.5rem;
+}
+
+.home-link .home-link-arrow {
+    width: 10px;
+    margin: 10px auto;
+    border-top: 35px solid white;
```
Comment (mporam) [inline styles.css]: hex colors

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,475 @@
...
+    border-left: 20px solid transparent;
+    border-right: 20px solid transparent;
+}
+
+#about {
+    background-color: #FFFFFF;
```
Comment (mporam) [inline styles.css]: hex code capitalisation consistency. use all upper or all lower throughout file

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,475 @@
...
+    margin: 170px auto 100px auto;
+    overflow: auto;
+    padding: 20px;
+}
+
+#about main img {
+    float: left;
+    height: 150px;
+    width: 150px;
+    border: 2px solid black;
```
Comment (mporam) [inline styles.css]: hex

File: styles.css [hunk 11]
```diff
@@ -0,0 +1,475 @@
...
+    border-radius: 50%;
+    margin: 20px;
+}
+
+#about main p {
+    margin: 20px 0;
+    font-size: 1.1rem;
+    color: white;
+    text-shadow: 1px 1px black, -1px -1px black,
```
Comment (mporam) [inline styles.css]: hex

File: styles.css [hunk 12]
```diff
@@ -0,0 +1,475 @@
...
+
+/* Portfolio */
+
+#portfolio {
+    background-image: url('assets/projects-bg.jpg');
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+
+#portfolio main {
```
Comment (mporam) [inline styles.css]: some of these properties are the same as the main on #about main, generalise these into a single main selector

File: styles.css [hunk 13]
```diff
@@ -0,0 +1,475 @@
...
+    width: 90%;
+    background-color: rgba(0, 0, 0, 0.7);
+    border-radius: 10px;
+    margin: 170px auto 100px auto;
+    overflow: auto;
+    padding: 20px;
+}
+
+#portfolio .projects-container {
```
Comment (mporam) [inline styles.css]: this is similar to the main properties, maybe create a class for the generic styles

File: styles.css [hunk 14]
```diff
@@ -0,0 +1,475 @@
...
+    background-color: rgba(0, 0, 0, 0.7);
+    border-radius: 10px;
+    margin: 20px auto;
+    overflow: auto;
+    padding: 20px;
+}
+
+#portfolio .projects-container img {
+    width: 90%;
+    height: auto;
```
Comment (mporam) [inline styles.css]: this is default value for images, u can remove

File: styles.css [hunk 15]
```diff
@@ -0,0 +1,475 @@
...
+    margin: 0 auto;
+}
+
+#portfolio .projects-container h3 {
+    margin: 5px;
+}
+
+[class*="project-"] {
```
Comment (mporam) [inline styles.css]: why use different numbered classes if they are all the same styles? just use one class. could probably put this on the col- class inside #portfolio

File: styles.css [hunk 16]
```diff
@@ -0,0 +1,475 @@
...
+    min-height: 40vh;
+    background-image: url('assets/map.png');
+    background-size: cover;
+}
+
+#contact .details {
+    margin: 10px 0 0 0;
+    width: 30%;
+    padding: 0 20px 10px 20px;
+    border: 1px solid black;
```
Comment (mporam) [inline styles.css]: hex

File: styles.css [hunk 17]
```diff
@@ -0,0 +1,475 @@
...
+    #about .badges {
+        margin: 20px auto 30px auto;
+    }
+
+    #portfolio main {
+        margin: 60px auto;
+    }
+
+    #contact .details {
+        margin: 0 0 0 0;
```
Comment (mporam) [inline styles.css]: just set this to margin: 0;

File: styles.css [hunk 18]
```diff
@@ -0,0 +1,475 @@
...
+}
+
+#contact .details ul {
+    padding: 0;
+    width: 100%;
+    margin: 0 auto;
+}
+
+#contact .details li {
+    display: block;
```
Comment (mporam) [inline styles.css]: this line does nothing. li elements are already block and floated elements become block.

File: styles.css [hunk 19]
```diff
@@ -0,0 +1,475 @@
...
+    #portfolio main {
+        margin: 60px auto;
+    }
+
+    #contact .details {
+        margin: 0 0 0 0;
+    }
+
+    #contact .map {
+        margin: 0 0 0 0;
```
Comment (mporam) [inline styles.css]: see above

File: styles.css [hunk 20]
```diff
@@ -0,0 +1,475 @@
...
+        margin: 50px auto 30px auto;
+
+    }
+
+    #about main p {
+        margin: 30px 0 10px 0;
+    }
+
+    #contact .details {
+        margin: 0 0 0 0;
```
Comment (mporam) [inline styles.css]: margin: 0;

File: styles.css [hunk 21]
```diff
@@ -0,0 +1,475 @@
...
+    }
+
+    #contact .map {
+        margin: 0 0 0 0;
```
Comment (mporam) [inline styles.css]: margin: 0;

File: styles.css [hunk 22]
```diff
@@ -0,0 +1,475 @@
...
+
+    nav .nav-list {
+        position: static;
+        float: none;
+        margin: 0;
+        padding: 0;
+    }
+
+    .nav-list li {
+        display: block;
```
Comment (mporam) [inline styles.css]: this does nothing.

File: index.html [hunk 23]
```diff
@@ -0,0 +1,153 @@
...
+                        <img src="assets/github.png" alt="github logo">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/nick-santiago-blair-21baa45a" target="_blank">
+                        <img src="assets/linkedin.png" alt="linkedin logo">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://www.google.com/gmail/" target="_blank">
```
Comment (mporam) [inline index.html]: lookup mailto: links, that would be better for this

File: index.html [hunk 24]
```diff
@@ -0,0 +1,153 @@
...
+        </div>
+        <a href="#about">
+        <div class="home-link">
+            Click Here!!
+            <div class="home-link-arrow"></div>
+        </div>
+        </a>
+    </section>
+    <section id="about">
+        <main>
```
[the remark this answers is not in the document]
Comment (nsantiagoblair) [reply index.html]: `<section>` identifies the position on the page according to the navigation bar and `<main>` contains the written content of the `<section>`. `<main>` also acts as a container with a different `background-color`.

File: index.html [hunk 25]
```diff
@@ -0,0 +1,153 @@
...
+            <div class="col-4">
+                <img  class="js" src="assets/js.png" alt="javascript logo">
+            </div>
+            <div class="col-4">
+                <img src="assets/ruby.png" alt="ruby logo">
+            </div>
+        </div>
+    </section>
+    <section id="portfolio">
+        <main>
```
[the remark this answers is not in the document]
Comment (nsantiagoblair) [reply index.html]: same as above

File: index.html [hunk 26]
```diff
@@ -0,0 +1,153 @@
...
+        <a class="site-logo" href="#"><h1>伲呵</h1></a>
+        <ul class="nav-list">
+            <li><a href="#">Home</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#contact">Contact</a></li>
+        </ul>
+    </nav>
+    <section class="home">
+            <video loop muted autoplay class="bg-video">
```
[the remark this answers is not in the document]
Comment (charliecog) [reply index.html]: ?

File: index.html [hunk 27]
```diff
@@ -0,0 +1,153 @@
...
+                tempus tellus vitae, mollis orci.
+                Cras blandit velit id risus finibus volutpat.
+                Nunc facilisis fermentum ex in cursus.
+                Nam non neque ut lacus egestas semper.
+                Morbi ut nisl et ligula congue pellentesque.
+                Nam bibendum aliquet eleifend. Fusce.
+            </p>
+        </div>
+        <a href="#about">
+        <div class="home-link">
```
[the remark this answers is not in the document]
Comment (charliecog) [reply index.html]: ?

File: index.html (hunk 24 above)
[the remark this answers is not in the document]
Comment (charliecog) [reply index.html]: function of main agreed with @nsantiagoblair however semantics of using the main tag multiple places on same page not ok. recommend leaving around section and changing other mains to a div or something suitable

File: index.html (hunk 25 above)
[the remark this answers is not in the document]
Comment (charliecog) [reply index.html]: see above response

File: styles.css (hunk 1 above)
Comment (charliecog) [reply styles.css]: ?

File: styles.css (hunk 9 above)
Comment (charliecog) [reply styles.css]: ?

File: styles.css (hunk 12 above)
Comment (charliecog) [reply styles.css]: ?

File: styles.css (hunk 22 above)
Comment (charliecog) [reply styles.css]: ?
li is already a block level element

Comment (nsantiagoblair) [reply styles.css]: it was made `inline-block` and this line is turning it back to `block` so navbar elements collapse for mobile

File: index.html [hunk 28]
```diff
@@ -36,21 +36,19 @@ <h1>I'm a <span class="underline">full-stack</span> web developer .</h1>
...
-                Donec at condimentum justo. Suspendisse turpis libero, euismod lacinia ipsum at, varius fringilla sem.<br><br>
+            <p>Avid traveller, aspiring developer and impelling tinkerer.
+                After a following at full-stack web development bootcamp at the
+                <a href="https://maydenacademy.co.uk/">Mayden Academy</a>
+                I have become fascinated with development at large. I can build
+                <span class="highlights">interactive front ends</span>
+                and
+                <span class="highlights">high-performance and secure CMS systems </span>
+                using a variety of technologies.
+                <br><br>
```
Comment (charliecog) [inline index.html]: ewwwwww `<br>` tags

File: index.html [hunk 29]
```diff
@@ -36,21 +36,19 @@ <h1>I'm a <span class="underline">full-stack</span> web developer .</h1>
...
+                When I’m not on my computer I enjoy trekking, cycling, photography and playing my guitar.
+                <br><br><br>
```
Comment (charliecog) [inline index.html]: shame on you

File: index.html [hunk 30]
```diff
@@ -36,21 +36,19 @@ <h1>I'm a <span class="underline">full-stack</span> web developer .</h1>
...
+                Feel free to contact me and we can work out how to <a href="#contact">make your project a reality!</a>
+                <br><br>
```
Comment (charliecog) [inline index.html]: yuk

Comment (nsantiagoblair) [reply index.html]: so mean! 😿

**Pull request merged**